ams/server/src/main/java/com/netease/arctic/server/dashboard/MixedAndIcebergTableDescriptor.java

```diff
@@ -364,14 +358,14 @@ public List<PartitionBaseInfo> getTablePartitions(AmoroTable<?> amoroTable) {
       return new ArrayList<>();
     }
     Map<String, PartitionBaseInfo> partitionBaseInfoHashMap = new HashMap<>();
-    getTableFiles(amoroTable, null)
+    getTableFiles(amoroTable, null, null)
```

Comment:
Can the method `getTableFiles` be changed to use an iterator instead of caching all files in a list to reduce memory overhead?